strings.js

```diff
@@ -0,0 +1,37 @@
+// 1. Uppercase the first character
+
+function ucFirst(str) {
+    return str[0].toUpperCase() + str.slice(1).toLowerCase();
+}
+
+let text = 'teXt';
+console.log(ucFirst(text));
+
+// 2. Check for spam
+
+function checkSpam(str) {
+    const LowerCaseStr = str.toLowerCase();
+
+    return LowerCaseStr.includes('viagra') || LowerCaseStr.includes('XXX'.toLowerCase())
```

Comment:
no need to convert 'XXX' to lower case, just write 'xxx'

Reply:
ok)